test/Bet.noblock.test.js

```diff
@@ -473,3 +474,32 @@ test('Test Weird Jonas Case', async () => {
 
   expect(await bet.calcSellFromAmount(989886n, 0)).toBe(490099n);
 });
+
+test('Initial Quote Prices set correctly', () => {
+  const bet = new Bet('test-bet', 4);
+  const price = bet.calcInitialPrice();
+  expect(price).toEqual(0.25);
+});
+
+test('Converts price to decimal', () => {
+  const bet = new Bet('test-bet', 4);
```

Comment:
do not use same bet id twice. you have dependency on the previous test here. the nanoid you use below is the way to go

Reply:
I have updated thanks